test/getConfig.js

```diff
@@ -20,24 +20,11 @@ test('read config from package.json', (t) => {
   let packageJson = json.readToObjSync(path.join(cwd, 'package.json'));
   packageJson.sgc = sgcrc;
 
-  // copy package.json to package.json.back
-  Promise.resolve(fs.createReadStream(cwd + '/package.json')
-    .pipe(fs.createWriteStream(cwd + '/package.json.back')))
-    .then(() => {
-      // delete package.json
-      fs.unlink(cwd + '/package.json');
-    })
-    .then(() => {
-      // write new package.json with the sgc property
-      fs.writeFile(cwd + '/package.json', JSON.stringify(packageJson), () => {
-        // perform test
-        t.deepEqual(getConfig(), sgcrc);
-      });
-    }).then(() => {
-      // delete package.json
-      fs.unlink(cwd + '/package.json');
-    }).then(() => {
-      // restore package.json from package.json.back
-      fs.rename(cwd + '/package.json.back', cwd + '/package.json');
-    });
+  fs.createReadStream(cwd + '/package.json')
+    .pipe(fs.createWriteStream(cwd + '/package.json.back'));
+  fs.unlinkSync(cwd + '/package.json');
+  fs.writeFileSync(cwd + '/package.json', JSON.stringify(packageJson));
+  t.deepEqual(getConfig(), sgcrc);
+  fs.unlinkSync(cwd + '/package.json');
+  fs.renameSync(cwd + '/package.json.back', cwd + '/package.json');
```

Comment:
I totally forgot to change the directory for the linting. You might get some errors - can you change the directory `tests` in `(package.json).scripts.lint` to `test`?

from
```js
"lint": "eslint lib tests",
```
to 
```js
"lint": "eslint lib test",
```